prdoc/pr_7200.prdoc

```diff
@@ -0,0 +1,38 @@
+title: 'XCM: Deny barrier checks for nested XCMs with specific instructions to be
+  executed on the local chain'
+doc:
+- audience: Runtime User
```

Comment:
```suggestion
- audience: Runtime Dev
```

Only runtime devs care about the xcm configuration. Also, devs should update theirs if needed.